### PR TITLE
Improvements of format_entries

### PR DESCRIPTION
**Pull request by sermir2003**
Hello! First of all, I would like to thank you for your wonderful repository with exercises. You have done a great job.

However, I found a function `python/exercises/practice/ledger/ledger.py:format_entries` in the repository whose code seemed unnecessarily complicated to me. I would like to try to make it easier, this is what this pull-request is dedicated to.

I managed to make some things easier. Although I was faced with a choice whether to use the new `locale` library to delegate currency rendering and get a generally accepted format, or it is better to format it with my hands the way it was accepted in the repository before. If you are interested in this improvement, please let me know your preferred method. They both have advantages and disadvantages.

In the end, I have to admit that I am a beginner in open source development. Perhaps this pull-request will fail the tests, in fact, so far, it is only in development stage, I just wanted to suggest an improvement and ask your opinion about the `locale` library and formatting style.

### CODE REVIEW

**Comment by sermir2003** [comment]
OK, you're not interested right now:/ But if you want to simplify this or any other code, please contact me, I will be ready to do it! :)

File: exercises/practice/ledger/ledger.py [hunk 1]
```diff
@@ -17,283 +19,80 @@ def create_entry(date, description, change):
...
+    )
+
+    entries.sort(key=lambda x: (x.date, x.change, x.description))
+    for entry in entries:
+        table.append('\n')
+
+        # Find next entry in order
+        entry = min(entries, )
+        entries.pop(entry)
+
```
Comment (sermir2003) [inline exercises/practice/ledger/ledger.py:67]: Actually, I have to delete these four lines

File: exercises/practice/ledger/ledger.py [hunk 2]
```diff
@@ -17,283 +19,80 @@ def create_entry(date, description, change):
...
+                )
+            )
+        table.append(' | ')
+
+        # Write entry change to table
+        change_str = locale.currency(entry.change / 100)
+        # WARNING: generally speaking, it is not worth using floating-point numbers when working with currency.
+        # It is possible to get rid of them here, but the code will be a little less elegant.
+        # Perhaps the fix will be in the next commit.
+        table.append('{: >13}'.format(change_str))
```
Comment (sermir2003) [inline exercises/practice/ledger/ledger.py:96]: And here I also should use column size variables

Comment (IsaacG) [reply exercises/practice/ledger/ledger.py:96]: I think you may be misunderstanding (1) this exercise and (2) the Exercism workflow. 

This exercise is a refactoring exercise. The goal of the exercise is to simplify the code. This code is intentionally overly complicated.

**Comment by BethanyG** [comment]
@sermir2003  -- Please read the message that was posted when this issue was closed. 

1.  This repo is closed to community contributions.  If you want a discussion or to propose a PR, use the [forum](forum.exercsim.org)
2.  This repo is not the place to solve exercises.  You use `exercism.org` for that.
3.  Creating noise by adding to an already closed issue  when someone doesn't respond to you is not welcomed nor appreciated.

@IsaacG   -- Please don't respond to already closed issues.  Thanks!

**Pull request closed without merging**